### PR TITLE
Add placeholder to SearchScreen text field and focus on when launch screen

### PR DESCRIPTION
**Pull request by iiinaiii**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Focus on Textfield when SearchScreen is launched. Keyboard for input is appear when this timing.
- Add placeholder to SearchScreen Textfield.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/d8e41fe5-e40b-4678-8606-4254784e44e1" width="300" > | <video src="https://github.com/user-attachments/assets/3ac17846-c1fc-4eef-975e-1e65825307a6" width="300" >

### CODE REVIEW

**Comment by iiinaiii** [comment]
As for the wording of the Placeholder, it was “Search” on iOS, so I made it the same as that.
I thought it would be better to use the phrase “Search Session”, what do you think ?

**Comment by takahirom** [comment]
"Search sessions/セッションを検索" sounds great. Could you change it? 🙏

**Comment by iiinaiii** [comment]
@takahirom 
> "Search sessions/セッションを検索" sounds great. Could you change it? 🙏

OK!
I can also make corrections on the iOS side, but is it okay to handle it in this pull request?

**Comment by takahirom** [comment]
You can do it in this PR. But separated pull request would be better because there are different reviewers.

**Comment by iiinaiii** [comment]
Understood.
I will only modify the Android side in this pull request

**Comment by iiinaiii** [comment]
I modified the words of placeholder.

en | ja
:--: | :--:
 |

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/SearchTextFieldAppBar.kt [hunk 1]
```diff
@@ -56,7 +75,32 @@ fun SearchTextFieldAppBar(
                         focusManager.clearFocus()
                     },
                 ),
+                decorationBox = { innerTextField ->
+                    TextFieldDefaults.DecorationBox(
```
Comment (takahirom) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/SearchTextFieldAppBar.kt:79]: DecorationBox 📝

**Review by takahirom — APPROVED** [review]
Looks great! Thank you for your contribution!

**Comment by iiinaiii** [comment]
Sorry, I forgot to write the #ISSUE_NUMBER of close in the PR.
The issue number is #992 .

Please close the issue.

**Pull request merged**